src/components/eligibility/Eligibility.tsx

```diff
@@ -76,6 +66,47 @@ export const Eligibility: React.FunctionComponent<any> = (props: any) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [step])
 
+  useEffect(() => {
+    const steps = {
```

Comment:
Nice and descriptive!

only one suggestion; can we move this to our constants?

Reply:
moved.